components/form/fields/IconSelectField.tsx

```diff
@@ -0,0 +1,39 @@
+import { InputLabel, MenuItem, Select } from '@mui/material'
```

Comment:
So this is either only ever gonna be used on the Social Icons component, or it can be made reusable. Either direction is an extremely easy change.

1. If only ever used for the Footer -> Contact icons on the dashboard, move this component there.
2. If reusable: Create a prop called `menuItems` that is an array, pass in `SocialMenuItems` as a value to this component from where it's being called (dashboard footer widget).